TRADEMARKS.md

```diff
@@ -1,76 +1,90 @@
+* Use that does not comply with the terms of this Policy.
 
-Amazon recognizes that community members writing related software need some way to identify the Amazon product to which that software pertains.
-Users should not be confused as to whether the related software is an official Amazon product or otherwise approved by Amazon.
-The name of such software may not include, in whole or in part, the Bottlerocket Trademarks in a way that suggests a connection between Amazon and the extension.
+You may not use the Bottlerocket Trademarks in connection with use or distribution of the Bottlerocket software, except as permitted by this Policy.
 
-## QUESTIONS
+## Questions
 
-If you are unsure whether your use of the Bottlerocket Trademarks would run afoul of this policy, feel free to contact us and ask.
+If you are unsure whether your use of the Bottlerocket Trademarks is permitted under this Policy, feel free to contact us and ask.
 If you have questions about these guidelines or use of any other Amazon trademark, please contact trademarks@amazon.com for assistance, or write to us at:
 
-**Amazon.com, Inc.<br>
-Attention: Trademarks<br>
-PO Box 81226<br>
-Seattle, WA 98108-1226**
+```text
+Amazon.com, Inc.
+Attention: Trademarks
+PO Box 81226
+Seattle, WA 98108-1226
+```
 
-This policy is based in part on the [open source trademark policy defined by the Mozilla organization](https://www.mozilla.org/en-US/foundation/trademarks/policy/), as adapted to [Amazon’s Carbonado policy](https://github.com/Carbonado/Carbonado/blob/master/TRADEMARK.md), as adapted to [Amazon’s Corretto policy](https://github.com/corretto/corretto-8/blob/master/TRADEMARKS.md).
-Thus, the text of this policy (and not the Bottlerocket Trademarks themselves) is licensed under the Creative Commons “Attribution-ShareAlike 2.0” license.
+This Policy is based in part on the [open source trademark policy defined by the Mozilla organization](https://www.mozilla.org/en-US/foundation/trademarks/policy/), therefore, the text of this Policy (and not the Bottlerocket Trademarks themselves) is licensed under the Creative Commons "Attribution-ShareAlike 2.0" license.
 
-## FAQ:
+## FAQ
 
 **1. Can I create and redistribute my own builds of Bottlerocket?**
-If you build Bottlerocket from unmodified source and redistribute the results, you may use “Bottlerocket” only if it is clear in both the name of your distribution and the content associated with it that your distribution is your build of Amazon’s Bottlerocket and not the official build, and you must identify the commit from which it is built, including the commit date.
+If you build Bottlerocket from unmodified source and redistribute the results, you may use "Bottlerocket" only if it is clear in both the name of your distribution and the content associated with it that your distribution is your build of Bottlerocket and not the official build, and you must identify the commit from which it is built, including the commit date.
```

Comment:
Might be good to add a line break after these questions to clearly separate them from the answers.